install-e2e-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+echo "Running npm install..."
+cd e2e
+npm install
+cd ../keep-ecdsa/solidity
+npm install --save-dev sha3
```

Comment:
If this dependency is required it should be included in keep-ecdsa's `package.json`, we should not need to install it manually.

Reply:
The manual install of `sha3` is needed for successful running of `npm link` in `local-setup/keep-ecdsa/solidity`. I tried replacing `npm install --save-dev sha3` with `npm rebuild`, but it did not help for the problem. I brought back the original solution.